addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4815,13 +4824,30 @@ registry.ImageOptimize = ImageHandlerOption.extend({
         const img = this._getImg();
         const match = img.src.match(/\/web_editor\/image_shape\/(\w+\.\w+)/);
         if (img.dataset.shape && match) {
-            await this._loadImageInfo(`/web/image/${match[1]}`);
+            return this._loadImageInfo(`/web/image/${match[1]}`);
```

Comment:
Mmh in fact, kinda concerned that `willStart` calls `_initializeImage` which modifies the DOM. DOM modifications should not be done in `willStart` (ideally, probably saving those info in some options class instance's variables and only applying them on the DOM when an option is applied is probably a better idea).

But let's not worry about that now, it was like that long before the shape on image feature. Let's merge :+1: